### ISSUE TITLE
Proposed changes to the gene search and trait association search specs

### ISSUE THREAD

**Issue by alancleary** [legumeinfo/website-ui-specs#18]
The trait association web component that will soon be released has the ability to have its genus and species selectors pre-set to specific values. The release will also add the ability to pre-set the species to the gene search component. Is there any objection to adding these features to the trait association search and gene search specs and bumping their versions to 1.2.0 and 1.2.0, respectively?

**Edit:** I think we should also change the name of the trait association search's spec directory from `trait-search` to `trait-association-search` and similarly rename the file within. This will better align it with the contents of the spec and the web component that implements it.

**Comment by StevenCannon-USDA** [comment]
Sounds good to me. Let me know if you'd like me to make those changes (since I authored the spec).

**Comment by StevenCannon-USDA** [comment]
I've gone ahead and made these changes. @alancleary - let me know if further tweaking is needed.

**Comment by alancleary** [comment]
Thanks @StevenCannon-USDA. The trait association search spec explicitly mentioned the "current" version of the spec in a couple places, which became inaccurate when you bumped the version so I went ahead and removed specific version numbers from the text of the spec. I also updated the gene search spec.

Let me know if you catch anything else. I can tag releases for these when people think they're ready.
